ddl/mock.go

```diff
@@ -148,43 +148,48 @@ func NewMockStateSyncer() syncer.StateSyncer {
 	return &MockStateSyncer{}
 }
 
+// clusterState mocks cluster state.
+// We move it from MockStateSyncer to here. Because we want to make it unaffected by ddl close.
+var clusterState *atomicutil.Pointer[syncer.StateInfo]
```

Comment:
will this clusterState changed in concurrency way？

Reply:
It's possible(call `UpdateGlobalState` in parallel), so it's used `atomicutil`.